src/cmd/reconcile.go

```diff
@@ -30,7 +29,6 @@ var reconcileCmd = &cobra.Command{
 		queue := make(chan opslevel_jq_parser.ServiceRegistration, 1)
 		common.SetupControllers(config, queue, resync)
 		common.ReconcileServices(client, disableServiceCreation, queue)
-		opslevel_common.Run("Controller")
```

Comment:
Should `common.Run("Controller")` be here?

Reply:
@davidbloss the signal code setup for this cobra subcommand is here: https://github.com/OpsLevel/kubectl-opslevel/pull/239